_install-and-configure/upgrade-opensearch/appendix/rolling-upgrade-lab.md

```diff
@@ -0,0 +1,887 @@
+   ecommerce                              0 r STARTED  4675   3.9mb 172.20.0.13 os-node-03
+   .kibana_1                              0 p STARTED     3   5.9kb 172.20.0.12 os-node-02
+   .kibana_1                              0 r STARTED     3   5.9kb 172.20.0.11 os-node-01
+   .kibana_92668751_admin_1               0 p STARTED    33  37.3kb 172.20.0.13 os-node-03
+   .kibana_92668751_admin_1               0 r STARTED    33  37.3kb 172.20.0.11 os-node-01
+   opensearch_dashboards_sample_data_logs 0 p STARTED 14074   9.1mb 172.20.0.12 os-node-02
+   opensearch_dashboards_sample_data_logs 0 r STARTED 14074   9.1mb 172.20.0.14 os-node-04
+   security-auditlog-2023.02.28           0 p STARTED     6  26.2kb 172.20.0.11 os-node-01
+   security-auditlog-2023.02.28           0 r STARTED     6  26.2kb 172.20.0.14 os-node-04
+   .opendistro-reports-definitions        0 p STARTED     0    208b 172.20.0.12 os-node-02
+   .opendistro-reports-definitions        0 r STARTED     0    208b 172.20.0.13 os-node-03
+   .opendistro-reports-definitions        0 r STARTED     0    208b 172.20.0.14 os-node-04
+   security-auditlog-2023.03.06           0 r STARTED    53 174.6kb 172.20.0.12 os-node-02
+   security-auditlog-2023.03.06           0 p STARTED    53 174.6kb 172.20.0.14 os-node-04
+   .kibana_101107607_newuser_1            0 r STARTED     1   5.1kb 172.20.0.13 os-node-03
+   .kibana_101107607_newuser_1            0 p STARTED     1   5.1kb 172.20.0.11 os-node-01
+   .opendistro_security                   0 r STARTED    10  64.5kb 172.20.0.12 os-node-02
+   .opendistro_security                   0 r STARTED    10  64.5kb 172.20.0.13 os-node-03
+   .opendistro_security                   0 r STARTED    10  64.5kb 172.20.0.11 os-node-01
+   .opendistro_security                   0 p STARTED    10  64.5kb 172.20.0.14 os-node-04
+   .kibana_-152937574_admintenant_1       0 r STARTED     1   5.1kb 172.20.0.12 os-node-02
+   .kibana_-152937574_admintenant_1       0 p STARTED     1   5.1kb 172.20.0.14 os-node-04
+   security-auditlog-2023.03.07           0 r STARTED    37 175.7kb 172.20.0.12 os-node-02
+   security-auditlog-2023.03.07           0 p STARTED    37 175.7kb 172.20.0.14 os-node-04
+   .kibana_92668751_admin_2               0 p STARTED    34  38.6kb 172.20.0.13 os-node-03
+   .kibana_92668751_admin_2               0 r STARTED    34  38.6kb 172.20.0.11 os-node-01
+   .kibana_2                              0 p STARTED     3     6kb 172.20.0.13 os-node-03
+   .kibana_2                              0 r STARTED     3     6kb 172.20.0.14 os-node-04
+   .opendistro-reports-instances          0 r STARTED     0    208b 172.20.0.12 os-node-02
+   .opendistro-reports-instances          0 r STARTED     0    208b 172.20.0.11 os-node-01
+   .opendistro-reports-instances          0 p STARTED     0    208b 172.20.0.14 os-node-04
+   ```
+
+### Checking data consistency
+
+Query the ecommerce index again to confirm that the sample data is still present.
```

Comment:
```suggestion
Query the ecommerce index again to confirm that the sample data is still present:
```

Reply:
This sentence reads as though it should be a numbered step and might flow better as a "you need to...in order to" statement.